GenderPayGap.WebUI/Controllers/ManageOrganisations/ManageOrganisationsController.cs

```diff
@@ -73,6 +65,7 @@ public IActionResult ManageOrganisationGet(string id)
             }

```

Comment:
(I've lost track of what's already on this branch/review/PR but) I think there's a helper for this (the "404 if decryption fails")